ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Node/ProtocolInfo.hs

```diff
@@ -35,7 +36,7 @@ enumCoreNodes (NumCoreNodes numNodes) =
 -- | Data required to run the specified protocol.
 data ProtocolInfo b = ProtocolInfo {
         pInfoConfig     :: TopLevelConfig b
-      , pInfoInitLedger :: ExtLedgerState b -- ^ At genesis
+      , pInfoInitLedger :: ExtLedgerState b ValuesMK -- ^ At genesis
```

Comment:
I think what the comment here means is that this holds the genesis ledger state which *could* or *should* contain UTXOs. Is that the case? Maybe we should make the comment a little more verbose